src/Microsoft.OData.Core/UriParser/Parsers/UriQueryExpressionParser.cs

```diff
@@ -822,8 +855,9 @@ private QueryToken ParsePrimaryStart()
 
                 case ExpressionTokenKind.Identifier:
                     {
-                        IdentifierTokenizer identifierTokenizer = new IdentifierTokenizer(this.parameters, new FunctionCallParser(this.lexer, this));
-                        return identifierTokenizer.ParseIdentifier(null);
+                        IdentifierTokenizer identifierTokenizer = new IdentifierTokenizer(this.parameters, this.FunctionCallParser);
+                        var parent = this.aggregateExpressionParents.Count > 0 ? this.aggregateExpressionParents.Peek() : null;

```

Comment:
>var parent = thi [](start = 24, length = 16)

we would like to steer clear of var so as to provide clarity around types used/expected.